index.bs

```diff
@@ -1866,6 +1902,28 @@ To <dfn>parse report windows</dfn> given a |value|, a
     1. Set |startDuration| to |endDuration|.
 1. Return |windows|.
 
+To <dfn>parse summary buckets</dfn> given a [=map=] |map|:
+1. Let |values| be the trivial mapping «[ 1, 2, ..., MAX_UINT32 ]».
```

Comment:
Can we use an [inclusive range](https://infra.spec.whatwg.org/#the-range) for this instead?